### ISSUE TITLE
Launching YAUML locally is not robust

### ISSUE THREAD

**Issue by ablondin** [yauml/yauml#12]
When testing YAUML locally, we need to call the script `process.py` from the directory `bin`. It should be callable from any repository.

Moreover, we should rename `process.py` by `yauml` since it is the main script. I also dislike what the Makefile does. When you type `make`, it should create the `bin/yauml` desired file, but in my cas, it does nothing:

    sed -e "s,template_filename.*=.*'.*',template_filename = '/usr/share/yauml/template.dot'," bin/process.py \
		>bin/yauml

When I change the end of the line, though, it works:

    sed -e "s,template_filename.*=.*'.*',template_filename = '/usr/share/yauml/template.dot'," bin/process.py >bin/yauml

I think we should separate clearly `make` and `make install`. In fact, `make` is useless, since there is nothing to compile locally. However, `make install` should do as a first step what `make` actually does.

**Comment by ablondin** [comment]
@sim590 : what do you think?

**Comment by sim590** [comment]
> I think we should separate clearly make and make install. In fact, make is useless, since there is nothing to compile locally. However, make install should do as a first step what make actually does.

Yes, you're right. The first rule (`config`) in the Makefile is only relevant when installing.

> Moreover, we should rename process.py by yauml since it is the main script.

If so, the `sed` call should replaced with `sed -i` for modifying the file "in place".

> When I change the end of the line, though, it works

 That's odd. I thought that this use of backslashes should be syntactically correct as mentionned [here](https://www.gnu.org/software/make/manual/html_node/Splitting-Lines.html). I'm normally trying to keep lines shorter than 80 or 100 characters. Have you tried to find out if trailing white space could be the issue?

**Comment by sim590** [comment]
> When testing YAUML locally, we need to call the script process.py from the directory bin. It should be callable from any repository.

Yes. Therefore, an absolute path has to be given.
